internal/controller/promotion/git_test.go

```diff
@@ -466,16 +470,15 @@ func TestMoveRepoContents(t *testing.T) {
 	const subdirCount = 50
 	const fileCount = 50
 	// Create dummy repo dir
-	srcDir, err := createDummyRepoDir(subdirCount, fileCount)
+	srcDir, err := createDummyRepoDir(t, subdirCount, fileCount)
 	defer os.RemoveAll(srcDir)
```

Comment:
With `t.TempDir()` you no longer need the defer (see #1677)

Reply:
I know. Not sure how I missed it though. Thanks for spotting.

Reply:
Fixed in https://github.com/akuity/kargo/pull/1674/commits/f1131668df6d29abd60a690865e754d9d73db46c